tutorials/singularity-containers-with-cloud-build/index.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Building Singularity Containers using Cloud Build
+description: Learn how to use Cloud Build to build Singularity Containers for HPC workloads
+author: wardharold
+tags: Cloud Build, Singularity, HPC
+date_published: 2019-01-18
+---
```

Comment:
I think it would be good to have a diagram here.  Make a simple crappy one and then ask the graphics team to fix it. They are awesome.